modules/claims_api/config/schemas/v2/generate_pdf_526.json

```diff
@@ -100,9 +100,8 @@
             "state": {
               "description": "State for the Veteran's current mailing address.",
               "type": "string",
-              "example": "OR",
-              "maxLength": 1000,
-              "minLength": 0
```

Comment:
Did you mean to remove the `minLength`? Should the max be 2 because of upstream?

Reply:
I mean to remove it, the max length is enforced by the regex so in line with the other schemas I just did not note a max length since 2 is all this will take always.  But that was my understanding for what we wanted for the docs here.  

@klwatanabe If the max characters accepted s 2, via the REGEX pattern, do we want to still mark that as maxLength: 2 in the documentation?  I did not see it happening anywhere else  so thought it might be worth asking here to make sure this gets totally addressed in this ticket.